iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -173,9 +171,14 @@ export class Inspector {
             // messages to itself when exporting files.
             return;
         }
-
-        if (data.task == 'SHOW_FACT') {
-            this.selectItem(data.factId);
+        if (String(data.task) === SHOW_FACT) {
+            const factId = String(data.factId);
+            const vuid = this.parseVuid(factId, true);
```

Comment:
It seems like we should add `data.docSetId` instead of speculatively parsing the data factId.

@joyreistad-wf Can you open a ticket for us to make a change to our internal service using API?

Reply:
Sure!

Reply:
The goal with this is to assume that if we're receiving a fact ID in the legacy format (`f-id1`) that the API user doesn't expect multi-doc support, so defaulting to a report index of 0 should be fine. 

The scenario of `{"docSetId": 0, "factId": "f-id1"}`, doesn't make much sense because `factId` is still in a format we don't intend to continue supporting. 

I'd think we do either `{"docSetId": 0, "factId": "id1"}` and have ixbrl-viewer generate the VUID based off of that, or just do `{"factId": "f0-id1"}`. I'm fine either way, but the latter does not require another API update.

Reply:
AFAIK, SHOW_FACT expects plain XML IDs (`"id1"`) rather than ixbrl-viewer-qualified (`"f-id1"`); at least, I'm not aware of code in our internal service that uses SHOW_FACT that would be prepending that `f-`.  It seems like the simplest bug fix is to add just add `viewerUniqueId(0, str)`, and then we can separately figure out the contract for multi-IXDS.

For multi-IXDS, I prefer the `{"docSetId":0,"factId":id1"}` rather than needing to couple to the VUID format in callers, but I guess I don't feel too strongly.

Reply:
Ah, I see. That was a misunderstanding on my part that the `f-id1` format was being used. In that case I agree with the `docSetId` approach.